rpmlb/recipe.py

```diff
@@ -138,3 +252,13 @@ def make_sequence(name: str, count: int):
         sequence_map = dict(starmap(make_sequence, count_map.items()))
 
         return sequence_map
+
+
+class RecipeError(Exception):
+    """A class to manage validation error for recipe data."""
+
+    messages = {}
```

Comment:
@khardix thanks for the mentioning it. I did know that.
I will fix to use a instance attribute.